### ISSUE TITLE
Key inputs doesnt fully work

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#766]
**Behavior of the program**
When watching a vid in fullscreen and u press the left right up down buttons it doenst do anything. Only after i click somewhere in the vid (for example i tap my trackpad to pause) so the player gets its first input the keys start working.

**Expected behavior**
Keys work without the player having the need of a first input

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Go to watch any video u like
2. Click on the fullscreen button
3. Try to use the Keys and u will see it doesnt work 
4. Click somewhere in the video to make it pause the video
5. Try the keys and u'll that now they are working

**Screenshots**
None

**Desktop (please complete the following information):**
 - OS: [Win 10 Home]
 - OS Version: [1909]
 - FreeTube version: [0.9.2.]
 - FreeTube build: [Release]

**Additional context**
None

**Comment by Synkky** [comment]
Related to #637

**Comment by hollunder** [comment]
Another way to reproduce this is to focus FreeTube using keyboard.
Under Windows that would probably be done using ALT+TAB.
Note that this also seems to be working on and off...

**Comment by hollunder** [comment]
I guess this has something to do with keyboard vs. mouse focus and maybe keyboard controls in general.

A related annoyance is that often in fullscreen mode the space key does not pause playback but rather exits fullscreen. If that happens you can also enter fullscreen by pressing space again. I guess there might be a keyboard focus on the fullscreen toggle button.

I found that if I hit TAB in that situation then space toggles playback, which is consistent with the order you can cycle through the controls by pressing TAB. Likewise, SHIFT+TAB lets freetube toggle playback on SPACE again. The only difference is that after hitting TAB the controls in question are highlighted, which they are initially not.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Closing in favor of https://github.com/FreeTubeApp/FreeTube/issues/4137

Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/4137
